main.typ

```diff
@@ -95,8 +95,13 @@ Additionally, Agile projects are designed to deliver working software in short i
 Despite these differences, Agile and Open Source management styles share some important similarities. Both approaches value flexibility, adaptability, and iteration. They also both emphasize the importance of collaboration and communication. As a result, these two management styles can be complementary, and they can be used together to achieve great success.
 
 == The Impact of Agile Methods on Open Source Development
-As Agile Methodologies integrate into mainstream development due to the positive effects on code quality and quick response time, aspects of agile are emerging into open source development. A trend in open source projects appear to be the adoption of Testing and Continuous Integration. The implementation of automated tests establishes trust among the developers involved with the open source project because passing tests demonstrates that their code works, and that the code does not have to be rewritten by other developers. @Turnu_2006
-// TODO: expand on this, cite maye 3-4 different more sources (check out Coram_2005)
+As Agile Methodologies integrate into mainstream development due to the positive effects on code quality and quick response time, aspects of agile are emerging into open source development. A trend in open source projects appear to be the adoption of Testing and Continuous Integration. The implementation of automated tests establishes trust among the developers involved with the open source project because passing tests demonstrates that their code works, and that the code does not have to be rewritten by other developers. @Turnu_2006 
+
+In addition to Testing and CI, other Agile practices are also making their way into open source development. Iterative development models are becoming more common, allowing features to be developed, tested, and released in small increments. This approach aligns with the Agile principle of responding quickly to changes and adapting to user feedback. Moreover, collaborative coding and pair programming, inspired by Agile's emphasis on teamwork, are being embraced to improve code quality and foster knowledge sharing within the open source community. This not only helps in maintaining code quality but also in mentoring new contributors​​. @Hron_2022
```

Comment:
> Moreover, collaborative coding and pair programming,

We mention that open source development is mostly asynchronous, so stuff like pair programming coming to open source doesn't really make sense.